pkg/nodejs/packagejson/parse_test.go

```diff
@@ -0,0 +1,119 @@
+package packagejson
```

Comment:
```suggestion
package packagejson_test
```

Reply:
For all other unit tests in the project, we did not use similar package naming. I was expecting naming to be in sync.